src/main/app/idam/authorizationMiddleware.ts

```diff
@@ -16,7 +16,7 @@ const logger = require('@hmcts/nodejs-logging').getLogger('middleware/authorizat
  * But most likely if we get 401 or 403 then the user's token has expired
  * So make them login again
  */
-function hasTokenExpired (err) {
+export function hasTokenExpired (err) {
   return (err.statusCode === HttpStatus.FORBIDDEN || err.statusCode === HttpStatus.UNAUTHORIZED)
```

Comment:
Will this always mean that the token is expired?

Reply:
Is it possible that we'll have a situation where e.g. a user doesn't have a required role and this piece of code will tell that his token has expired (while it hasn't)?

Reply:
so the scenario you provided wouldn't happen

Reply:
👍